src/gp_activetable.c

```diff
@@ -236,7 +248,10 @@ report_altered_reloid(Oid reloid)
 	if (IsRoleMirror() || IS_QUERY_DISPATCHER()) return;
 
 	LWLockAcquire(diskquota_locks.altered_reloid_cache_lock, LW_EXCLUSIVE);
-	hash_search(altered_reloid_cache, &reloid, HASH_ENTER, NULL);
+	HASHACTION action =
```

Comment:
Is it possible to mix declaration and code in a diskquota?

Reply:
this is already used in a lot of places in diskquota.